components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManager.java

```diff
@@ -281,4 +281,14 @@ int getRelativeDepthBetweenOrganizationsInSameBranch(String firstOrgId, String s
      * @throws OrganizationManagementServerException The server exception thrown when fetching the primary organization.
      */
     String getPrimaryOrganizationId(String organizationId) throws OrganizationManagementServerException;
+
+    /**
+     * Add Root organization which can initiate an organization tree.
+     *
+     * @param tenantId The tenant ID corresponds to the root organization.
+     * @param organization The organization to be created.
+     * @return The newly created organization.
+     * @throws OrganizationManagementException The exception thrown when creating an organization.
+     */
+    Organization addRootOrganization(int tenantId, Organization organization) throws OrganizationManagementException;
```

Comment:
need to be a default interface method

Reply:
Fixed by - https://github.com/wso2/identity-organization-management-core/pull/89/commits/9c30415a2ea1cfc93bd23a06f09254730216aee9